### PR TITLE
Largecardmain/add to garden btn

### PR DESCRIPTION
**Pull request by Raiza-D**

### CODE REVIEW

File: server/routes/my_garden.js [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
     db.query(getUserPlantInfo(req.params.id)).then((data) => {
-      console.log(data);
       res.json(data.rows);
     });
   });
 
   router.delete("/:id", (req, res) => {
     db.query(deleteUserPlant(req.params.id)).then(() => {
       res.send();
-      // Do I need to send anything here?
```
Comment (MatiasWengiel) [inline server/routes/my_garden.js:20]: The server should send a status code of success (I forget the number) after a successful delete call

File: src/components/LargeCardMain.jsx [hunk 2]
```diff
@@ -19,6 +21,22 @@ export default function LargeCardMain(props) {
...
+
+  const addPlantToGarden = () => {
+    const info = {
+      userId: 1,
+      plantId: id
+    }
+
+    axios.post("/api/my_garden/submit", info).then((response) => {
+      console.log(response.data);
+      navigate("/my_garden");
```
Comment (MatiasWengiel) [inline src/components/LargeCardMain.jsx:31]: Do we still need this console log of the data?

**Review by MatiasWengiel — DISMISSED** [review]
Awesome work! I left a couple of comments. They aren't strictly necessary for this PR, so I'll approve it and you can merge it whenever you're ready, but take a look and let me know what you think! 

I'm also trying out the review comments thing here on GitHub, so let me know how this worked please!

File: server/routes/my_garden.js (hunk 1 above)
Comment (Raiza-D) [reply server/routes/my_garden.js:20]: Status 200 added

File: src/components/LargeCardMain.jsx (hunk 2 above)
Comment (Raiza-D) [reply src/components/LargeCardMain.jsx:31]: Removed

**Review by caitlincroteau — APPROVED** [review]

**Pull request merged**